mitosheet/mitosheet/public/v3/types/int.py

```diff
@@ -18,6 +21,9 @@ def cast_to_int(unknown: Union[str, int, float, bool, datetime, timedelta]) -> O
     elif isinstance(unknown, int):
         return unknown
     elif isinstance(unknown, float):
+        if np.isnan(unknown):
```

Comment:
Instead of handling casting errors like this, what do you think about wrapping `cast_to_int` in a try, catch?

Reply:
That's so much better!